.ci/static-checks.sh

```diff
@@ -20,19 +20,60 @@ script_name=${0##*/}
 repo=""
 master_branch="false"
 
+typeset -A long_options
+
+long_options=(
+	[commits]="Check commits"
+	[docs]="Check document files"
+	[files]="Check files"
+	[golang]="Check '.go' files"
+	[help]="Display usage statement"
+	[licenses]="Check licenses"
+	[master]="Force checking of master branch"
+	[repo:]="Specify GitHub URL of repo to use (github.com/user/repo)"
+	[versions]="Check versions files"
+)
+
 usage()
 {
 	cat <<EOT
 
 Usage: $script_name help
-       $script_name repo-name [true]
+       $script_name [options] repo-name [true]
+
```

Comment:
Somewhere, but I don't know where, we should probably try and note in the help that if you do not supply any options, then the default is 'all'. I don't think that is listed anywhere in the help?

Reply:
Done.